### ISSUE TITLE
birthdays are doubled

### ISSUE THREAD

**Issue by gg-gg1** [SimpleMobileTools/Simple-Calendar#1677]
Hi. I have noticed a strange behaviour when I tick the add new birthdays automatically option that it keeps adding my contacts birthdays again and again. If I untick the option all looks ok.

**Comment by tibbi** [comment]
what app and OS version are you using?

**Comment by gg-gg1** [comment]
> what app and OS version are you using?

crDroid and simple calendar. I sync with the aosp contacts in which I add birthdays from Simple Contacts as aosp contacts doesn`t have the add birthday option.

**Comment by tibbi** [comment]
I mean the number of Simple Calendars version, at the bottom of About section

**Comment by gg-gg1** [comment]
> I mean the number of Simple Calendars version, at the bottom of About section

Version 6.17.4 Pro

**Comment by Naveen3Singh** [comment]
@gg-gg1 

**Q.** What contacts app are you using?

**Q.** Is it possible that your contacts are duplicated across multiple accounts? (e.g. two google accounts, sim card and/or your phone)

> crDroid

it's a good ROM :)

**Comment by EDavidF** [comment]
Hi - is it possible you have a synced calendar that is something like "Contact Birthdays" but also clicked the "Add Contact Birthdays" from the menu?

**Comment by tibbi** [comment]
likely related to #1658

**Comment by tibbi** [comment]
not sure, which ones are doubled? If we had some reliable way of reproducing the issue, that would help. Like which app you use for creating the contacts? Do those birthdays have a year too, or not?

**Comment by gg-gg1** [comment]
Hi. It doesn`t matter which contacts app I am using, the birthdays are shown doubled if I activate the add birthdays automatically function. The birthdays have year too, yes. But it doubles also the one without year. I will attach a screenshot of how it looks like.
Thanks.

![simple calendar](https://user-images.githubusercontent.com/98748676/179764642-ff464212-8b2d-42f8-bf2f-5be92f0a24c3.png)

**Comment by tibbi** [comment]
have you maybe changed your system language or renamed the Birthdays event type?

**Comment by gg-gg1** [comment]
> @gg-gg1
> 
> **Q.** ~What contacts app are you using?~ _Already answered above. My bad._
> 
> **Q.** Is it possible that your contacts are duplicated across multiple accounts? (e.g. two google accounts, sim card and/or your phone)
> 
> > crDroid
> 
> it's a good ROM :)

I don`t know. By that do you mean if I enter in Contacts and select a contact if it shows send sms, call on Whatsapp and others, yes, it shows.

**Comment by gg-gg1** [comment]
> have you maybe changed your system language or renamed the Birthdays event type?

I am using english language all time, never change.
The Birthdays event type is the same. I just select "Add contact birthdays" from the menu and tick "Add new birthdays automatically"... It doubles them only if I tick that option. If I don`t tick, all works fine and no double entries.

**Comment by amadicus** [comment]
Magic 4.0 Android 10. If you leave a checkmark "add new birthdays automatically" when adding birthdays, then after some time the birthdays in the calendar are duplicated.  If you uncheck the box, everything is fine.

**Comment by tibbi** [comment]
duplication seems to be caused by WhatsApp. If anyone checks Simple Contacts Pros contact View details screen, is there WhatsApp shown at the bottom as one of the contact sources?

**Comment by amadicus** [comment]
I caught a bug with doubling the birthdays in the calendar when I did not have WhatsApp installed.  The bug appears if you check the "add new birthdays automatically" checkbox.  If it is not set, birthdays are not duplicated.  But I checked your recommendation.  Screenshot is attached.
![Screenshot_20221207_053926_com.simplemobiletools.contacts.pro.jpg](https://user-images.githubusercontent.com/101167707/206091822-1ae25434-354c-403a-8a07-46599a946f64.jpg)
![Screenshot_20221207_055435_com.simplemobiletools.contacts.pro.jpg](https://user-images.githubusercontent.com/101167707/206091999-d38d6086-01a2-40fd-b239-d698d02ed96e.jpg)
